lib/OpenQA/Schema/ResultSet/Assets.pm

```diff
@@ -273,6 +282,25 @@ END_SQL
         $group_info{$largest_group}->{picked} += $size;
     }
 
+    # produce cache file for /admin/assets
+    my $cache_file_path = status_cache_file;
+    try {
+        my $cache_file = Mojo::File->new($cache_file_path);
+        # ensure parent directory exists
+        $cache_file->dirname->make_path();
```

Comment:
you can't create that file as geekotest in production - you need to create this in the package as writeable

Reply:
Seems like the directory is already packages - but it belongs to the worker package. So I better choose another path, maybe `$OPENQA_BASEDIR/openqa/webui/cache`?